src/lib/containers/SynapseTable.js

```diff
@@ -129,39 +140,57 @@ export default class SynapseTable extends React.Component {
-                    </tbody>
-                </table>
-                {pastZero && <button onClick={this.handlePaginationClick(PREVIOUS)} className="btn btn-default" style={{borderRadius: "8px", color: "#1e7098", background: "white"}} type="button">
-                    Previous
-                </button>}
-                <button onClick={this.handlePaginationClick(NEXT)} className="btn btn-default" style={{borderRadius: "8px", color: "#1e7098", background: "white"}} type="button">
-                    Next
-                </button>
-            </div>
+            <React.Fragment>
+                <div className="container">
+                    <div className="row">
+                        <span>
+                            <strong>Showing 2530 files</strong> 
+                        </span>
+                        <a onClick={this.advancedSearch} href="" className="floatRight">
+                            <u> Advanced Search </u>
+                        </a>
+                        <span className="floatRight">&nbsp;&nbsp;</span>
+                        <a onClick={this.download} href="" className="floatRight">
+                            <u>Download</u>
+                        </a>
+                    </div>
+                </div>
+                <div className="container overflowAuto">
+                    <div className="row">
+                        <table className="table table-striped table-condensed">
+                            {/* show the column headers */}
+                            <thead>
+                                <tr>
+                                    <th></th>
+                                    {columnModels.map(
+                                        column => {
+                                            return (<th key={column.name}>
+                                                        <a onClick={this.handleColumnClick(column.name)} className="padding-left-2 padding-right-2" > {column.name}
+                                                            <i className="fa"></i>
+                                                        </a>
+                                                    </th>)
+                                        }
+                                    )}
+                                </tr>
+                            </thead>
+                            {/* show the actual table body */}
+                            <tbody>
+                                {rowsFormatted.map(
+                                    rowFormatted => {
+                                        return rowFormatted
+                                    }
+                                )}
+                            </tbody>
+                        </table>
+                        {pastZero && <button onClick={this.handlePaginationClick(PREVIOUS)} className="btn btn-default" style={{borderRadius: "8px", color: "#1e7098", background: "white"}} type="button">
```

Comment:
I'd go through the codebase and replace any inline styles with css classes (so they can be overridden by dev users of the components)